config.js

```diff
@@ -1,32 +1,34 @@
 const babelParse = require("@babel/parser/new/index").parse;
 const acornParse = require("acorn").parse;
 const esprimaParse = require("esprima").parse;
-const cherowParse = require("cherow").parse;
+const meriyahParse = require("meriyah").parse;
 
 exports.files = [
   "./fixtures/es5/angular.js",
   "./fixtures/es5/ember.debug.js",
   "./fixtures/es5/babylon-dist.js",
   "./fixtures/es5/jquery.js",
   "./fixtures/es5/backbone.js",
-  "./fixtures/es5/react-with-addons.js"
+  "./fixtures/es5/react-with-addons.js",
+  "./fixtures/es6/angular-compiler.js",
+  "./fixtures/es6/material-ui-core.js",
 ];
 
 exports.parsers = {
   acorn: {
     parse: acornParse,
-    options: { sourceType: "script" }
+    options: { sourceType: "module" }
   },
   babel: {
     parse: babelParse,
-    options: { sourceType: "script" }
-  },
-  cherow: {
-    parse: cherowParse,
-    options: { sourceType: "script" }
+    options: { sourceType: "module" }
   },
   esprima: {
     parse: esprimaParse,
-    options: { sourceType: "script" }
-  }
+    options: { sourceType: "module" }
+  },
+  meriyah: {
```

Comment:
@JLHwung You don't need to set any options for this. `parse` is legacy and belong to ES5.  `parseScript` and `parseModule` is in the ECMA specs now, so use `const meriyahParse = require("meriyah").parse;Module` and skip the options to avoid overhead.